SPOSite: Renaming in DSC is failing but can complete successfully in SPO Admin UI

#### Details of the scenario you tried and the problem that is occurring
Having created a new SPSite resource in M365DSC I have attempted to change the name of the site and then apply the updated configuration. When the configuration goes through the Release Pipeline (using the AZDO Whitepaper) I get an error that the PnP module cannot perform the operation.

If I were to rename the SPO Site in SharePoint admin I don't get an issue and this completes successfully

#### Verbose logs showing the problem
```
2022-11-03T12:40:47.9457530Z VERBOSE: [vmdevops001]: LCM:  [ Start  Resource ]  [[SPOSite]ConfigureTestSite]
2022-11-03T12:40:47.9669049Z VERBOSE: [vmdevops001]: LCM:  [ Start  Test     ]  [[SPOSite]ConfigureTestSite]
2022-11-03T12:40:48.0330394Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Testing configuration for site collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:48.4393471Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Getting site collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:49.6428013Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Current Values: AllowSelfServiceUpgrade=True; AnonymousLinkExpirationInDays=0; ApplicationId=***; ApplicationSecret=$null; CertificatePassword=$null; CertificatePath=***; CertificateThumbprint=; Classification=$null; CommentsOnSitePagesDisabled=False; Credential=***; DefaultLinkPermission=None; DefaultSharingLinkType=None; DenyAddAndCustomizePages=True; DisableAppViews=NotDisabled; DisableCompanyWideSharingLinks=NotDisabled; DisableFlows=False; DisableSharingForNonOwners=$null; Ensure=Present; HubUrl=$null; LocaleId=1033; LogoFilePath=$null; Managedidentity=False; OverrideTenantAnonymousLinkExpirationPolicy=False; Owner=arthur.curry@domain.com; RestrictedToRegion=; SharingAllowedDomainList=; SharingBlockedDomainList=; SharingCapability=Disabled; SharingDomainRestrictionMode=; ShowPeoplePickerSuggestionsForGuestUsers=False; SiteDesign=$null; SocialBarOnSitePagesDisabled=False; StorageMaximumLevel=26214400; StorageWarningLevel=25574400; Template=STS#3; TenantId=***; TimeZoneId=13; Title=Renamed Site3; Url=https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:49.6578611Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Target Values: CommentsOnSitePagesDisabled=False; DefaultLinkPermission=None; DefaultSharingLinkType=None; DisableAppViews=NotDisabled; DisableCompanyWideSharingLinks=NotDisabled; DisableFlows=False; Ensure=Present; LocaleId=1033; Owner=arthur.curry@domain.com; SharingCapability=Disabled; StorageMaximumLevel=26214400; StorageWarningLevel=25574400; Template=STS#3; TimeZoneId=13; Title=Matteos Site; Url=https://mojodevl.sharepoint.com/sites/testsite1; Verbose=True
2022-11-03T12:40:49.6658589Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Test-TargetResource returned False
2022-11-03T12:40:49.6666593Z VERBOSE: [vmdevops001]: LCM:  [ End    Test     ]  [[SPOSite]ConfigureTestSite]  in 1.6720 seconds.
2022-11-03T12:40:49.6696602Z VERBOSE: [vmdevops001]: LCM:  [ Start  Set      ]  [[SPOSite]ConfigureTestSite]
2022-11-03T12:40:49.6703855Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Setting configuration for site collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:50.1890892Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Getting site collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:50.6245298Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Site {https://mojodevl.sharepoint.com/sites/testsite1} already exists, updating its settings
2022-11-03T12:40:52.5499936Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Updating props via Set-PNPSite on https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T12:40:53.0834471Z VERBOSE: [vmdevops001]: LCM:  [ End    Set      ]  [[SPOSite]ConfigureTestSite]  in 3.4410 seconds.
2022-11-03T12:40:53.2833266Z ##[error]PowerShell DSC resource MSFT_SPOSite  failed to execute Set-TargetResource functionality with error message: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The requested operation is not supported for site: https://mojodevl-admin.sharepoint.com/ 
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : localhost
```

#### Suggested solution to the issue
N/A

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
        SPOSite 'ConfigureTestSite' {
            Url                            = "https://mojodevl.sharepoint.com/sites/testsite1"
            StorageMaximumLevel            = 26214400
            LocaleId                       = 1033
            Template                       = "STS#3"
            Owner                          = "arthur.curry@domain.com"
            Title                          = "Matteos Site"
            TimeZoneId                     = 13
            StorageWarningLevel            = 25574400
            SharingCapability              = "Disabled"
            CommentsOnSitePagesDisabled    = $false
            DisableAppViews                = "NotDisabled"
            DisableCompanyWideSharingLinks = "NotDisabled"
            DisableFlows                   = $false
            DefaultSharingLinkType         = "None"
            DefaultLinkPermission          = "None"
            Ensure                         = "Present"
            Credential                     = $credsGlobalAdmin
        }
```

#### The operating system the target node is running
OsName               : Microsoft Windows Server 2022 Datacenter Azure Edition
OsOperatingSystemSKU : 407
OsArchitecture       : 64-bit
WindowsVersion       : 2009
WindowsBuildLabEx    : 20348.1.amd64fre.fe_release.210507-1500
OsLanguage           : en-US
OsMuiLanguages       : {en-US, en-GB}

#### Version of the DSC module that was used ('dev' if using current dev branch)
1.22.1102.1

This is the log file from when the dsc deploy pipeline ran that errored (should have posted before the manual bit above

```
2022-11-03T16:25:53.8234250Z VERBOSE: [vmdevops001]: LCM:  [ Start  Resource ]  [[SPOSite]ConfigureTestSite]
2022-11-03T16:25:53.8452094Z VERBOSE: [vmdevops001]: LCM:  [ Start  Test     ]  [[SPOSite]ConfigureTestSite]
2022-11-03T16:25:53.9115919Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Testing configuration for site 
2022-11-03T16:25:53.9129061Z collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:54.2717670Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Getting site collection 
2022-11-03T16:25:54.2730676Z https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:55.3097006Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Current Values: 
2022-11-03T16:25:55.3119683Z AllowSelfServiceUpgrade=True; AnonymousLinkExpirationInDays=0; ApplicationId=***; ApplicationSecret=$null; 
2022-11-03T16:25:55.3128498Z CertificatePassword=$null; CertificatePath=***; CertificateThumbprint=; Classification=$null; 
2022-11-03T16:25:55.3145979Z CommentsOnSitePagesDisabled=False; Credential=***; DefaultLinkPermission=None; DefaultSharingLinkType=None; 
2022-11-03T16:25:55.3154054Z DenyAddAndCustomizePages=True; DisableAppViews=NotDisabled; DisableCompanyWideSharingLinks=NotDisabled; 
2022-11-03T16:25:55.3172163Z DisableFlows=False; DisableSharingForNonOwners=$null; Ensure=Present; HubUrl=$null; LocaleId=1033; LogoFilePath=$null; 
2022-11-03T16:25:55.3180532Z Managedidentity=False; OverrideTenantAnonymousLinkExpirationPolicy=False; Owner=arthur.curry@domain.com; 
2022-11-03T16:25:55.3202445Z RestrictedToRegion=; SharingAllowedDomainList=; SharingBlockedDomainList=; SharingCapability=Disabled; 
2022-11-03T16:25:55.3210901Z SharingDomainRestrictionMode=; ShowPeoplePickerSuggestionsForGuestUsers=False; SiteDesign=$null; 
2022-11-03T16:25:55.3229210Z SocialBarOnSitePagesDisabled=False; StorageMaximumLevel=2621440; StorageWarningLevel=2557440; Template=STS#3; 
2022-11-03T16:25:55.3247775Z TenantId=***; TimeZoneId=13; Title=Matt Wants a Title; Url=https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:55.3265211Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Target Values: 
2022-11-03T16:25:55.3274331Z CommentsOnSitePagesDisabled=False; DefaultLinkPermission=None; DefaultSharingLinkType=None; 
2022-11-03T16:25:55.3292277Z DisableAppViews=NotDisabled; DisableCompanyWideSharingLinks=NotDisabled; DisableFlows=False; Ensure=Present; 
2022-11-03T16:25:55.3300012Z LocaleId=1033; Owner=arthur.curry@domain.com; SharingCapability=Disabled; StorageMaximumLevel=26214400; 
2022-11-03T16:25:55.3413792Z StorageWarningLevel=25574400; Template=STS#3; TimeZoneId=13; Title=Matt Wants a Title; 
2022-11-03T16:25:55.3420441Z Url=https://mojodevl.sharepoint.com/sites/testsite1; Verbose=True
2022-11-03T16:25:55.3422219Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Test-TargetResource returned False
2022-11-03T16:25:55.3423576Z VERBOSE: [vmdevops001]: LCM:  [ End    Test     ]  [[SPOSite]ConfigureTestSite]  in 1.4720 seconds.
2022-11-03T16:25:55.3424441Z VERBOSE: [vmdevops001]: LCM:  [ Start  Set      ]  [[SPOSite]ConfigureTestSite]
2022-11-03T16:25:55.3425264Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Setting configuration for site 
2022-11-03T16:25:55.3426068Z collection https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:55.8231427Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Getting site collection 
2022-11-03T16:25:55.8243959Z https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:56.2921945Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Site 
2022-11-03T16:25:56.2934053Z {https://mojodevl.sharepoint.com/sites/testsite1} already exists, updating its settings
2022-11-03T16:25:58.4906228Z VERBOSE: [vmdevops001]:                            [[SPOSite]ConfigureTestSite] Updating props via Set-PNPSite on 
2022-11-03T16:25:58.4920125Z https://mojodevl.sharepoint.com/sites/testsite1
2022-11-03T16:25:58.9529885Z VERBOSE: [vmdevops001]: LCM:  [ End    Set      ]  [[SPOSite]ConfigureTestSite]  in 3.6410 seconds.
2022-11-03T16:25:59.1510883Z ##[error]PowerShell DSC resource MSFT_SPOSite  failed to execute Set-TargetResource functionality with error message: The 
running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: 
The requested operation is not supported for site: https://mojodevl-admin.sharepoint.com/ 
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
    + PSComputerName        : localhost
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -23,7 +23,8 @@
   * Fixed error with the export that wasn't properly returning the UpdateTimeOfDay if not in short time string format.
     FIXES [#2639](https://github.com/microsoft/Microsoft365DSC/issues/2639)
 * DEPENDENCIES
-  * Updated MSCloudLoginAssistant to version 1.0.99;
+  * Updated MSCloudLoginAssistant to version 1.0.100;
+    FIXES [#2484](https://github.com/microsoft/Microsoft365DSC/issues/2484)
 
 # 1.22.1207.1
 

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1
@@ -597,10 +597,9 @@ function Set-TargetResource
             $ConnectionMode = New-M365DSCConnection -Workload 'PnP' `
                 -InboundParameters $PSBoundParameters `
                 -Url $Url
-            Write-Verbose -Message "Updating props via Set-PNPSite on $($Url)"
+            Write-Verbose -Message "Updating props via Set-PNPSite on $($Url) with parameters:`r`n$(Convert-M365DscHashtableToString -Hashtable $UpdateParams)"
             Set-PnPSite @UpdateParams -ErrorAction Stop
         }
-
 
         $site = Get-PnPTenantSite $Url
 

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1
@@ -574,6 +574,7 @@ function Set-TargetResource
             AnonymousLinkExpirationInDays               = $AnonymousLinkExpirationInDays
             OverrideTenantAnonymousLinkExpirationPolicy = $OverrideTenantAnonymousLinkExpirationPolicy
             DenyAddAndCustomizePages                    = $deny
+            Title                                       = $Title
         }
         $UpdateParams = Remove-NullEntriesFromHashtable -Hash $UpdateParams
 

--- a/Modules/Microsoft365DSC/Dependencies/Manifest.psd1
+++ b/Modules/Microsoft365DSC/Dependencies/Manifest.psd1
@@ -74,7 +74,7 @@
         },
         @{
             ModuleName      = "MSCloudLoginAssistant"
-            RequiredVersion = "1.0.99"
+            RequiredVersion = "1.0.100"
         },
         @{
             ModuleName      = 'PnP.PowerShell'

